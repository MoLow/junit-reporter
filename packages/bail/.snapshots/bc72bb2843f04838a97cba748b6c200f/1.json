{
  "stderr": "node:internal/test_runner/harness:*\n      throw err;\n      ^\n\nError: Bail\n    *\n    at bail.next (<anonymous>)\n    *\n    *\n    *\n    *\n    *\n    *\n    *\n    *\nEmitted 'error' event on Duplex instance at:\n    *\n    *\n    *\n\nNode.js *\n",
  "stdout": "X\n\u001b[31mâœ– Bailing on failed test: fail\u001b[0m\n",
  "exitCode": 7
}