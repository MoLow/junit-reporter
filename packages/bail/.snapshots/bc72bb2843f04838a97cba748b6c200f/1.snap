Object {
  "exitCode": 7,
  "stderr": "node:internal/test_runner/harness:*
      throw err;
      ^

Error: Bail
    *
    at bail.next (<anonymous>)
    *
    *
    *
    *
    *
    *
    *
    *
Emitted 'error' event on Duplex instance at:
    *
    *
    *

Node.js *
",
  "stdout": "X
[31mâœ– Bailing on failed test: fail[0m
",
}